jfrog-cli/artifactory/cli.go

```diff
@@ -1060,7 +1060,12 @@ func getConfigFlags() []cli.Flag {
 			Usage: "[Default: true] If set to false then the configured password will not be encrypted using Artifatory's encryption API.",
 		},
 	}
-	return append(flags, getCommonFlags()...)
+	flags = append(flags, getBaseFlags()...)
+	return append(flags,
+		cli.StringFlag{
+			Name:  "ssh-key-path",
+			Usage: "[Optional] SSH key file path.",
+		})
 }
```

Comment:
Since the ssh-key-path flag is defined twice in this file, let's move its creation into a function and reuse it in these two places.